.github/workflows/build-packages.yml

```
@@ -34,17 +50,124 @@ jobs:
     runs-on: ubuntu-latest
+    outputs:
+      release_body: ${{ steps.generate-body.outputs.release_body }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Generate Release Body
+        id: generate-body
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2129:style:3:3: Consider using { cmd1; cmd2; } >> file instead of individual redirects

(shellcheck)